app/models/changelog.php

```diff
@@ -114,3 +108,27 @@
 
     return $link;
 };
+
+// Helper to generate change author names
```

Comment:
Yeah, and Travis doesn’t seem to complain. We’re not testing PHP 7 on Travis because that’s what we’re all using locally I guess?